packages/suite/src/components/suite/styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const CircleBorder = styled.div`
+    border: 9px solid ${({ theme }) => theme.backgroundPrimarySubtleOnElevation0};
+    background: ${({ theme }) => theme.backgroundPrimarySubtleOnElevation1};
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+export const Divider = styled.div`
```

Comment:
I think you use this component only once. Could you please move this component there?

Reply:
Actually it's used twice. In `StakingDashboard` and in `StakeEthCard`. These components are in different views: `wallet` and `dashboard` respectively. So it seems logical to have it in common sharable folder.